packages/@react-spectrum/datepicker/test/DatePicker.test.js

```diff
@@ -215,9 +215,10 @@ describe('DatePicker', function () {
       expect(onChange).toHaveBeenCalledTimes(1);
       expect(onChange).toHaveBeenCalledWith(new CalendarDate(2019, 2, 4));
       expect(getTextValue(combobox)).toBe('2/3/2019'); // controlled
+      await act(async () => Promise.resolve());
```

Comment:
All these extra promises in the tests worry me... why are they needed?

Reply:
these are cases where focus used to be lost, now it's being handled
the promise is to get the mutation observer to fire